README.md

```diff
@@ -102,9 +106,23 @@ These are the supported vector properties:
 
 #### points
 
-The `points` setting defines a list of points to be drawn on the
-board. It is optional. Each point should be defined as an object with
-`name` and `coords` properties.
+The `points` setting defines a list of points to be either just drawn
+on the board for reference, or to be placed by the student.  Points can
+have the following properties:
+
+* `name` (required): The unique name of the point, used in checks and
+  internally to identify the point.
+* `coords` (required): An array of length 2 with the (initial)
+  coordinates of the point.
+* `fixed`: Whether the point should be drawn in a fixed location or
+  placed by the student.  Defaults to `true`.
+* `render`: Whether the point should be rendered when the board is
+  initially shown.  Defaults to `true`, but may be set to `false` for
+  non-fixed points.
+* `description`: The long description used in the drop-down menu for
+  non-fixed points.
+* `style`: An object with addional properties for the point.  Supports
+  the properties `size`, `strokeColor` and `fillColor`.
```

Comment:
@smarnach nit: supporting both `strokeColor` and `fillColor` for "Point" seems a bit counter-intuitive for me. It might be more intuitive to have a `color` property that would expand into both `strokeColor` and `fillColor`.